storage/redis_adapter.go

```diff
@@ -29,11 +29,11 @@ type messagePipeliner struct {
 	errCh         chan error
 }
 
-func newMessagePipeliner(bufferSize int, redisClient *r.Client, errCh chan error) *messagePipeliner {
+func newMessagePipeliner(bufferSize int, redisClient *r.Client, timeoutSeconds int, errCh chan error) *messagePipeliner {
```

Comment:
Ah. Yes... I can manually convert if in the `parseConfig()` function. I didn't think of that before.